docs/submission/submitted-data.md

```diff
@@ -8,107 +8,90 @@ Program members can track the progress of their program’s clinical and molecul
 
-## Completed Core Clinical Data
+### Clinical Completeness Data Summary
 
-_Coming Soon:_ This line graph shows the number of donors that are clinically complete over time.
+This line graph compares how many donors have complete clinical data over time. 
+- Check out [Clinical Data Validation Rules](/docs/submission/clinical-data-validation-rules) for more information on how clinical completion is calculated.
+- As soon as a donors clinical data is completed, this chart will update in real-time. 
 
-## Molecular Data Summary
+### Molecular Data Summary
 
-_Coming Soon:_ This line graph compares how many donors have been processed through the [DNA-Seq Pipeline](/docs/analysis-workflows/dna-pipeline): Raw Read Submission, Alignment and Sanger VC.
+This line graph compares how many donors have been processed through the [DNA-Seq Pipeline](/docs/analysis-workflows/dna-pipeline) (Raw Read Submission, Alignment, Sanger VC...) over time.
+- Use the legend to toggle different pipeline steps on/off to check progress of different processing steps. 
+- This chart is kept up-to-date as data is processed in real-time.  While the data may be processed on the chart, it may not yet be released to the program if a donors is lacking complete clinical data. 
 
 ## Donor Data Summary
 
-This table is a detailed summary of the clinical data status and DNA-Seq pipeline status for each registered donor in the program. It also shows which donors have files that have been released to the public via the File Repository.
-![Dashbard Donor Data Summary](/assets/submission/dashboard-donor-table-card.png)
+This table is a detailed summary of the clinical data status and DNA-Seq pipeline status for **each registered donor** in the program. It also shows which donors have files that have been released to the public via the File Repository.
 
-#### Table Actions
+![Dashboard Workflow Status](/assets/submission/program_donor_summary_section.png)
 
-Using the buttons above the table, a program member can download all clinical data that has been submitted for the program donors. The download will be in the same file format that was submitted through the sample registration and clinical submission sections.
+### Table Actions
 
-_Coming Soon:_ A program member can download missing clinical data for all donors as well as the table summary details.
+Using the buttons above the table, a program member can download all clinical data that has been submitted for the program donors. The download will be in the same file format that was submitted through the sample registration and clinical submission sections.
+![Dashboard Workflow Status](/assets/submission/dashboard-workflow-statusbar.png)
```

Comment:
should this image be here?

Reply:
maybe this should be an image of the button?

Reply:
no removed